packages/dd-trace/test/opentracing/span_context.spec.js

```diff
@@ -68,6 +70,7 @@ describe('SpanContext', () => {
       _traceId: '123',
       _spanId: '456',
       _parentId: null,
+      _isRemote: null,
```

Comment:
Should default to `false` to avoid polymorphism.

Reply:
Good point. I got rid of the null and made the default value True to align with the python implementation.